### DISCUSSION TITLE
What is the pt_main_thread?

### DISCUSSION THREAD

**Discussion by ChenZhongPu** [open-webui/open-webui#3650] [Q&A]
I run `open-webui` using Docker, and I noticed a strange command `pt_main_thread` when invoking `top`:

```
4070215 root      20   0 6118488 601716 166748 S  43.8   0.5  10:39.92 pt_main_thread
```

It uses a significant CPU.

**Comment by Peter-De-Ath** [comment]
pytorch, if your using the default embedding engine. it'll use this in open-webui

**Reply by ChenZhongPu** [reply]
I also tried to use openai as the embedding model, but `pt_main_thread` is still running. Can I stop this command from running?

**Reply by Pranjal10052000** [reply]
@ChenZhongPu  were you able to fix this?
